azavea/pwd-fishway
# Select video in see the fishway

## Overview

Build out See the Fishway. Videos can be selected and autoplayed in the sidebar with accompanying information.

Connects #36 

### Demo
<img width="1158" alt="Screen Shot 2019-05-14 at 2 54 56 PM" src="https://user-images.githubusercontent.com/10568752/57724223-45195b00-7658-11e9-96c0-0450363355c6.png">

Styling super rough, obviously.

### Notes

I made a followup card to format the datetime, which was way more of a pain than I expected. Couldn't get Moment.js or Date.js working! 👎 #71 


## Testing Instructions
Netlify, visit See the Fishway tab.

Videos should autoplay. Custom controls to come.
